pkg/sloop/webserver/webserver.go

```diff
@@ -100,6 +103,34 @@ func webFileHandler(w http.ResponseWriter, r *http.Request) {
 	glog.V(2).Infof("webFileHandler successfully returned file %v for %v", fixedUrl, r.URL)
 }
 
+func backupHandler(db badgerwrap.DB, currentContext string) http.HandlerFunc {
```

Comment:
Please add a comment explaining the purpose (your PR has a great writeup, but it might not be obvious to someone reading the code)

Reply:
fixed